enterprise/fulltext-addon/src/test/java/org/neo4j/kernel/api/impl/fulltext/FulltextAnalyzerTest.java

```diff
@@ -67,11 +68,12 @@ public void shouldBeAbleToSpecifyEnglishAnalyzer() throws Exception
     @Test
     public void shouldBeAbleToSpecifySwedishAnalyzer() throws Exception
     {
+        analyzer = SWEDISH;
         try ( FulltextProvider provider = createProvider(); )
         {
-            FulltextFactory fulltextFactory = new FulltextFactory( fs, storeDir, SWEDISH, provider );
-            fulltextFactory.createFulltextIndex( BLOOM_NODES, NODES, singletonList( "prop" ) );
-            provider.init();
+//            FulltextFactory fulltextFactory = new FulltextFactory( fs, storeDir, SWEDISH, provider );
```

Comment:
Leftovers?

Reply:
Yeah, I'll box it up and put it in the fridge.